task_14/test/home.js

```diff
@@ -0,0 +1,55 @@
+const homepage = require('../pageobjects/homepage')
+var baseUrl = 'https://pastebin.com/'
+
+describe('I Can Win', function(){
+    this.timeout(5000)
+    it('Open https://pastebin.com', function(){
+        homepage.go_to_url(baseUrl)
+    })
+
+    it('Paste "Hello from WebDriver"', function(){
+        homepage.enter_paste('postform-text', 'Hello from WebDriver')
+    })
+
+    it('Paste Expiration: "10 Minutes"', function(){
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
    it("Paste Expiration: \"10 Minutes\"", function(){
```